libs/game/sprite.ts

```diff
@@ -135,6 +142,7 @@ class Sprite implements SpriteLike {
     setImage(img: Image) {
         if (!img) return; // don't break the sprite
         this._image = img;
+        this._hitboxes = game.calculateHitBoxes(this);
```

Comment:
What happens if someone does `mySprite.image.setPixel(10, 10, 12)` ? I guess we shouldn't support it....

Reply:
Yeah, doesn't work. I'd like to support it but I don't have any good ideas on how

Reply:
there's dirty flag on images, currently only used on `screen` - I guess we could use it, but maybe some time later